toolkit/tools/pkg/imagecustomizerlib/liveosisoconstants.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerlib
+
+const (
```

Comment:
Why not place these in `liveosisoutils.go` to be alongside `bootloaderFilesConfig`?

Reply:
fair - moved.